railties/test/application/test_runner_test.rb

```diff
@@ -727,6 +747,42 @@ def create_schema
         app_file "db/schema.rb", ""
       end
 
+      def create_test_for_env(env)
+        app_file "test/models/environment_test.rb", <<-RUBY
+          require 'test_helper'
+          class JSONReporter < Minitest::AbstractReporter
+            def record(result)
+              puts JSON.dump(klass: result.class.name,
+                             name: result.name,
+                             failures: result.failures,
+                             assertions: result.assertions,
+                             time: result.time)
+            end
+          end
+          
```

Comment:
There are trailing whitespaces here.

Reply:
Thanks!  I saw the complaints, but Vim didn't show for some reason.  😀